components/external_server/src/utils/functions.py

```diff
@@ -178,9 +178,9 @@ def md5_hash(string: str) -> str:
     return md5.hexdigest()
 
 
-def report_date_time() -> str:
+def report_date_time(attribute_name: str = "report_date") -> str:
     """Return the report date requested as query parameter if it's in the past, else return an empty string."""
-    if report_date_string := dict(bottle.request.query).get("report_date"):
+    if report_date_string := dict(bottle.request.query).get(attribute_name):
```

Comment:
shouldn't we parse frontend data to datetime and then write it back with isoformat(), to validate input?

Reply:
I guess that depends on whether we consider the API internal (i.e. just meant for the frontend) or if we also want to offer it to external consumers. I think at the moment we have only documented the API endpoints for export and import of reports, so all the other endpoints are _de facto_ for the frontend only.